src/App.jsx

```diff
@@ -1,13 +1,155 @@
-
 import './App.css'
+import React, {useState} from "react";
+import Button from "./components/Button/Button.jsx";
+
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [sum, setSum] = useState({
```

Comment:
De naam sum is niet veelzeggend, hier had je beter fruitState kunnen gebruiken en setFruitState.